### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by deiteris**
Hi guys, I've noticed this project and decided to contribute to the docs :) If you find this commit too large and hard to review, I can try splitting it on separate PRs to simplify review process. And thanks for the great project!

* Fix typos
* Correct the documentation styling
* Fix links to `.md` files
* Use title case capitalization in headings
* Remove punctuation characters from headings
* Add missing punctuation characters in lists, paragraphs, etc.
* Use auto-numbering for markdown lists
* Remove unnecessary empty lines
* Add C language identifier to code blocks in `defines_guide.md`
* Use consistent naming of owoTrack app
* Improve guide section names
* Improve wording of some procedures and steps

### CODE REVIEW

**Comment by Eirenliel** [comment]
Isn't .md links break when rendering on github pages?

**Comment by deiteris** [comment]
It shouldn't, as they're using Jekyll relative links plugin.
https://github.blog/2016-12-05-relative-links-for-github-pages/

**Comment by Eirenliel** [comment]
okay, then i'll wait for others' reasoning to why they were removed in the first place.

also good job :) are you in our discord?

**Comment by deiteris** [comment]
Yeah, I joined recently. My discord is Emojikage#3095.

**Comment by adigyran** [comment]
that's weird. links without md worked perfectly

**Review by adigyran — APPROVED** [review]
Everything is fine now, thanks for fixes :)

File: faq_owo.md [hunk 1]
```diff
@@ -1,141 +1,117 @@
...
-* [In what order should I start SteamVR and SlimeVR Server?](#in-what-order-should-i-start-steamvr-and-slimevr-server)
-* [My phone isn't connecting to the server](#my-phone-isnt-connecting-to-the-server)
-* [I have an Iphone and it disconnects after 10 minutes](#i-have-an-iphone-and-it-disconnects-after-10-minutes)
-* [Virtual trackers on steamVR are greyed out](#virtual-trackers-on-steamvr-are-greyed-out)
-* [I can't find Ios version in the store](#i-cant-find-ios-version-in-the-store)
-* [Other issues](#other-issues)
-
-## FAQ:
+owoTrack is a mobile application that can be used for VR tracking using your phone.
+The application uses SlimeVR Server for tracking. to set up SlimeVR Server, refer to [SlimeVR setup guide](slimevr-setup.md).
```
Comment (Eirenliel) [inline faq_owo.md:11]:
```suggestion
The application uses SlimeVR Server for tracking. To set up SlimeVR Server, refer to [SlimeVR setup guide](slimevr-setup.md).
```

File: faq_owo.md [hunk 2]
```diff
@@ -1,141 +1,117 @@
...
 
-### How many trackers or phones do I need to use for leg tracking?
+If you're using trackers/phones with owoTrack app for tracking there is minimum amount of 5 trackers/phones.
+For 1 phone use the [owoTrack SteamVR driver](https://github.com/abb128/owo-track-driver) instead.
```
Comment (Eirenliel) [inline faq_owo.md:14]:
```suggestion
```

File: faq_owo.md [hunk 3]
```diff
@@ -1,141 +1,117 @@
...
-* [Virtual trackers on steamVR are greyed out](#virtual-trackers-on-steamvr-are-greyed-out)
-* [I can't find Ios version in the store](#i-cant-find-ios-version-in-the-store)
-* [Other issues](#other-issues)
-
-## FAQ:
+owoTrack is a mobile application that can be used for VR tracking using your phone.
+The application uses SlimeVR Server for tracking. to set up SlimeVR Server, refer to [SlimeVR setup guide](slimevr-setup.md).
 
-### How many trackers or phones do I need to use for leg tracking?
+If you're using trackers/phones with owoTrack app for tracking there is minimum amount of 5 trackers/phones.
```
Comment (Eirenliel) [inline faq_owo.md:13]:
```suggestion
If you're using trackers/phones with owoTrack app for tracking there is minimum amount of 5 trackers/phones for full body tracking. One phone can be used if you want only wasit tracking. Waist tracking can also work with [owoTrack SteamVR driver](https://github.com/abb128/owo-track-driver), without SlimeVR.
```

File: index.md [hunk 4]
```diff
@@ -3,17 +3,16 @@ layout: page
 title: Home
 nav_order: 1
 ---
-# SlimeVR DIY GUIDE
 
-Welcome to the SlimeVR DIY guide. In this guide you will learn how to build your own SlimeVR trackers or use your mobile phones as SlimeVR trackers.
+# SlimeVR DIY guide
 
-**Please note you need at least 5 trackers (built boards or phones) for proper full body tracking.**
+Welcome to the SlimeVR DIY guide. In this guide you will learn how to build your own SlimeVR trackers or use your mobile phones as SlimeVR trackers.
```
Comment (Eirenliel) [inline index.md:9]:
```suggestion
Welcome to the SlimeVR guide. In this guide you will learn how to build your own SlimeVR trackers, setup the trackers you already have, or use your mobile phones as SlimeVR trackers.
```

File: slimevr-setup.md [hunk 5]
```diff
@@ -1,102 +1,77 @@
...
+{:toc}
 
-* Latest driver: [https://github.com/SlimeVR/SlimeVR-OpenVR-Driver/releases/latest/download/slimevr-openvr-driver-win64.zip](https://github.com/SlimeVR/SlimeVR-OpenVR-Driver/releases/latest/download/slimevr-openvr-driver-win64.zip)
-* Latest server: [https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR.zip](https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR.zip)
+## Readiness checklist
 
-**If something can't be resolved using those instructions, please contact on the [official SlimeVR discord](https://discord.gg/SlimeVR)  #technical-support channel for help!**
+* Download the latest [SlimeVR Driver](https://github.com/SlimeVR/SlimeVR-OpenVR-Driver/releases/latest/download/slimevr-openvr-driver-win64.zip).
+* Download the latest [SlimeVR Server](https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR.zip).
+* Turn each tracker on and see if it works.
```
Comment (Eirenliel) [inline slimevr-setup.md:24]:
```suggestion
* Turn each tracker on and see if they work.
```

File: slimevr-setup.md [hunk 6]
```diff
@@ -1,102 +1,77 @@
...
 
-Turn each tracker on and see if it works. Each tracker should light up a blue and an orange light briefly on startup (light colors and signaling are subject to change in the future!) Light will turn off after a second or so, it's normal. Trackers don't have LED indicator of being on, so don't forget to switch them off.
+Each tracker should light up a blue and an orange light briefly on startup (light colors and signaling are subject to change in the future!). Light will turn off after a second or so, it's normal. Trackers don't have LED indicator of being on, so don't forget to switch them off.
```
Comment (Eirenliel) [inline slimevr-setup.md:26]:
```suggestion
Each tracker should blink a LED briefly on startup, and then blink every 10-15 seconds once when the tracker is on. Any other lights (for example, always on, blinking multiple times, etc) mean something is wrong.
```

File: slimevr-setup.md [hunk 7]
```diff
@@ -1,102 +1,77 @@
...
 
 If a tracker don't start up, try charging it. Connect the tracker via USB port to your PC or any USB charger. Red LED light should light up to indicate that it's charging. Green LED light means it's fully charging. Try turning the tracker on during charging to see if it works.
 
-## Owotrack information
-
-**You can find full owoTrack FAQ [here](faq_owo).**
+## Before you start
 
-**If you're using phones with Owo app for tracking there is minimum amount of 5 phones. For 1 phone use official owo driver instead.**
+### Install beta version of SteamVR
```
Comment (Eirenliel) [inline slimevr-setup.md:32]:
```suggestion
### Install SteamVR
```

File: upload_firmware_guide.md [hunk 8]
```diff
@@ -1,93 +1,100 @@
...
 
-## Step Four: Install device drivers.
+## 4. Install device drivers
 
-You can download the drivers for windows for CP210X (NodeMCU v2) from silicon labs [here](https://www.silabs.com/documents/public/software/CP210x_Universal_Windows_Driver.zip). For any other os, downloads can be found [here](https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers). For CH430 (NodeMCU v3 and **review units of SlimeVR**), download from [here](https://cdn.sparkfun.com/assets/learn_tutorials/8/4/4/CH341SER.EXE).
+You can download the drivers for Windows for CP210X (NodeMCU v2) from silicon labs [here](https://www.silabs.com/documents/public/software/CP210x_Universal_Windows_Driver.zip).
+
+For any other OS, drivers can be found [here](https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers).
+
+For CH430 (NodeMCU v3 and **review units of SlimeVR**), download the drivers from [here](https://cdn.sparkfun.com/assets/learn_tutorials/8/4/4/CH341SER.EXE).
```
Comment (Eirenliel) [inline upload_firmware_guide.md:39]:
```suggestion
For CH340 (NodeMCU v3 and **review units of SlimeVR**), download the drivers from [here](https://cdn.sparkfun.com/assets/learn_tutorials/8/4/4/CH341SER.EXE).
```

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**